policy-commands.go

```diff
@@ -225,3 +225,164 @@ func (adm *AdminClient) SetPolicy(ctx context.Context, policyName, entityName st
 	}
 	return nil
 }
+
+// AttachPoliciesToUser - attach policies to a user.
+func (adm *AdminClient) AttachPoliciesToUser(ctx context.Context, policiesToAdd, accessKey string) error {
+	queryValues := url.Values{}
+	queryValues.Set("policiesToAdd", policiesToAdd)
+	queryValues.Set("accessKey", accessKey)
+
+	reqData := requestData{
+		relPath:     adminAPIPrefix + "/attach-user-policies",
+		queryValues: queryValues,
+	}
+
+	// Execute PUT on /minio/admin/v3/attach-group-policies
+	resp, err := adm.executeMethod(ctx, http.MethodPut, reqData)
+	defer closeResponse(resp)
+	if err != nil {
+		return err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return httpRespToErrorResponse(resp)
+	}
+	return nil
+}
+
+// DetachPoliciesFromUser - detach policies from a user.
+func (adm *AdminClient) DetachPoliciesFromUser(ctx context.Context, policiesToDetach, accessKey string) error {
+	queryValues := url.Values{}
+	queryValues.Set("policiesToDetach", policiesToDetach)
+	queryValues.Set("accessKey", accessKey)
+
+	reqData := requestData{
+		relPath:     adminAPIPrefix + "/detach-user-policies",
+		queryValues: queryValues,
+	}
+
+	// Execute PUT on /minio/admin/v3/attach-group-policies
+	resp, err := adm.executeMethod(ctx, http.MethodPut, reqData)
+	defer closeResponse(resp)
+	if err != nil {
+		return err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return httpRespToErrorResponse(resp)
+	}
+	return nil
+}
+
+// GetUserPolicies - get policies attached to a user
+func (adm *AdminClient) GetUserPolicies(ctx context.Context, name string) (p []string, err error) {
+	queryValues := url.Values{}
+	queryValues.Set("accessKey", name)
+
+	reqData := requestData{
+		relPath:     adminAPIPrefix + "/list-user-policies",
+		queryValues: queryValues,
+	}
+
+	// Execute GET on /minio/admin/v3/list-user-policies
+	resp, err := adm.executeMethod(ctx, http.MethodGet, reqData)
+
+	defer closeResponse(resp)
+	if err != nil {
+		return p, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return p, httpRespToErrorResponse(resp)
+	}
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return p, err
+	}
+
+	if err = json.Unmarshal(b, &p); err != nil {
+		return p, err
+	}
+
+	return p, nil
```

Comment:
Avoids using temporary buffer and doesn't use old ioutil as bonus.

```suggestion
	dec := json.NewDecoder(resp.Body)
	err = dec.Decode(&p)
	return p, err
```